### DISCUSSION TITLE
PBS script for a dp-train

### DISCUSSION THREAD

**Discussion by maverick422** [deepmodeling/deepmd-kit#1398] [Q&A] [answered]
Thanks for all!
I wrote a pbs job script but it always reports an error when I submit it , on a DeePMD-kit v2.0.3
here is the script 

`#!/bin/bash
#PBS -N dp-train
#PBS -l nodes=node008:ppn=2
#PBS -q gpu
#PBS -j oe

cd "$PBS_O_WORKDIR"
NPROCS=`wc -l < $PBS_NODEFILE`

CUDA_VISIBLE_DEVICES=0,1  horovodrun -np $NPROCS  dp train -in input.json >|LOG.dp 2>&1`

the log file is as attach file
[LOG.dp.txt](https://github.com/deepmodeling/deepmd-kit/files/7800929/LOG.dp.txt)
.

**Comment by njzjz** [comment, marked as answer]
`dp train -in input.json` is not correct. The correct command is `dp train input.json`.

**Reply by maverick422** [reply]
Thank you very much！
